MantidQt/CustomInterfaces/src/Tomography/TomographyIfaceModel.cpp

```diff
@@ -334,7 +328,99 @@ void TomographyIfaceModel::doQueryJobStatus(const std::string &compRes,
 }
 
 /**
- * Handle the job submission request relies on a submit algorithm.
+ * Build the components of the command line to run on the remote or local
+ * compute resource. Produces a (normally full) path to a runnable, and
+ * the options (quite like $0 and $* in scripts).
+ *
+ * @param comp Compute resource for which the command line is being prepared
+ * @param run Path to a runnable application (script, python module, etc.)
+ * @param opt Command line options to the application
+ */
+void TomographyIfaceModel::makeRunnableWithOptions(
+    const std::string &comp, std::string &run,
+    std::vector<std::string> &opt) const {
+
+  if (!m_currentToolSettings.get()) {
```

Comment:
I'm pretty sure that `if (m_currentToolSettings) {` does the same thing without getting the raw pointer